tests/ui/issues/issue-32782.rs

```diff
@@ -4,7 +4,7 @@ macro_rules! bar (
 
 macro_rules! foo (
     () => (
-        #[allow_internal_unstable] //~ ERROR allow_internal_unstable side-steps
+        #[allow_internal_unstable()] //~ ERROR allow_internal_unstable side-steps
```

Comment:
BREAKING CHANGE:
Previously, attributes on macro calls were partially checked for correct syntax (only what `check_builtin_meta_item` can check). 
After this PR, the attributes are fully parsed.

In any case, these attributes are always completely ignored, except for `#[cfg]`, which was always completely parsed and still is and is not affected by this breaking change.